### PR TITLE
AeroHIVE quickguide

### PR DESCRIPTION
**Pull request by whitx**
# Description

Added a quick guide for AeroHIVE configuration with PacketFence.
# Impacts

None
# Delete branch after merge

NO
# NEWS file entries
## Enhancements
- Added a quick guide for the configuration of AeroHIVE Access Point with PacketFence.

### CODE REVIEW

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
++++++++++++++++++++++++++++++++
+The last but not the least. Here, you start the PacketFence server according to the configurations made in the previous steps. If everything goes as expected, you'll be prompted by a window inviting you to continue to the web administration interface.
+
+You'll be able to login to the PacketFence web administration interface with the credentials created in Step 4.
+
+Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+
+Configuring AeroHIVE controller in PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now that you have a fully functionnal PacketFence installation, you'll need to add the AeroHIVE AP to the PacketFence switches database for correct intergration.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: integration

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+To do so, login to the PacketFence web administration interface if it is not already done. Click on the Configuration tab and select the switches section.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: Switches

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+
+We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
+
+Confirgure the switch like the following:
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: Configure

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+
+image::docs/images/aerohive-pf-config-role.png[scaledwidth="100%",alt="Roles switch settings"]
+
+image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
+
+Modify your RADIUS secret.
+
+Click Add switch
+
+The newly added controller should shows up in the list.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: show up

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+PacketFence configuration is done. You may now reboot the demonstation PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administation interface.
+
+Step 3: Configure AeroHIVE
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+This section will discuss about the configuration of your AeroHIVE AP in order to use it with our configured PacketFence environment.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: This section will help you configure your AeroHIVE AP for integration with PacketFence using the VLAN-based enforcement.

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+
+Connect APs and Controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+First thing to do is to connect the HiveManager. Like you saw in Step 2, make sure to connect the AP on the TRUNK port configured for it.
+
+Provision the HiveManager
+^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the HiveManager might be empty, add your AP on it via it's serial number. If necessra update the firmware of the HiveManager and the AP.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: its serial
necessary

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 7]
```diff
@@ -0,0 +1,301 @@
...
+
+NOTE: Please refer to the AeroHIVE Quick Devices Start Guide to add devices: http://docs.aerohive.com/330000/docs/help/english/documentation/AerohiveDevicesQuickStart.pdf
+
+NOTE: Please refer to the following guide to learn more about the HiveManager: http://docs.aerohive.com/330000/docs/help/english/documentation/Aerohive_HiveManagerNG_GettingStartedGuide.pdf
+
+Setup the SSIDs
+^^^^^^^^^^^^^^^
+For this guide, you will show you how to configure one open SSID with RADIUS mac filtering, and one secure SSID using WPA2-Enterprise.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: In this guide, we will configure one open SSI using RADIUS MAC-based filtering and one secured SSID using WPA2 Enterprise.

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 8]
```diff
@@ -0,0 +1,301 @@
...
+
+Login to your HiveManager using the admin credentials. You should see the Dashboard. Click on the Configure pane, and follow the following steps.
+
+Configure the AAA Servers
++++++++++++++++++++++++++
+On the left menu, click on Advanced Configuration, then Authentication, then AAA Client Settings. We need to add the PacketFence RADIUS server here. Click on the New button, and provide follow this configuration.
+
+image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+
+Click OK to save the server. You should see the RADIUS server in the list.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: Click Ok to save your changes.

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 9]
```diff
@@ -0,0 +1,301 @@
...
++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those information.
+
+image::docs/images/aerohive-ssid-public.png[scaledwidth="100%",alt="SSID open configuration"]
+
+Click OK to save the SSID.
+
+Configure the Secure SSID
++++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those inforamtion.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: and configure the SSID like in the following screen capture:

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 10]
```diff
@@ -0,0 +1,301 @@
...
+
+Create user profile as follow:
+
+image::docs/images/aerohive-user-profile-reg.png[scaledwidth="100%",alt="User profile registration"]
+
+image::docs/images/aerohive-user-profile-prod.png[scaledwidth="100%",alt="User profile production"]
+
+Configure SNMP
+^^^^^^^^^^^^^^
+You now need to configure the SNMP to add a valid read community string. Click on the Configure pane, and on Advanced Configuration, then Management Policies, then SNMP Assignements. Now click on New and follow this configuration.
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: Assignments

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 11]
```diff
@@ -0,0 +1,301 @@
...
+
+image::docs/images/aerohive-add-snmp.png[scaledwidth="100%",alt="SNMP configuration"]
+
+Configure RADIUS attribute
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: Attribute

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 12]
```diff
@@ -0,0 +1,301 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To be able to use the "Roles by switch role" you will need to setup a specific RADIUS attribute. To do so in the Configuration pane got to Networks, click on the network you will use, that will bring you to the "Guided configuration" now choose your SSID and click 'Add/Remove' under User Profile configure now this section as follow:
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: To do so, from the Configuration panel, click on Networks, ...

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 13]
```diff
@@ -0,0 +1,301 @@
...
+Validate all the modifications and click on close.
+
+Step 5: Test and Demonstrate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Congratulations, you have everything setup and ready!  If your setup is properly configured, you should be able to :
+
+* reach (ping) the controller from the PacketFence environment
+* see the DemoOpen, and DemoSecure SSIDs
+* login the PacketFence administrative UI (https://management_IP:1443)
+* connects a client device on the DemoOpen SSID using demouser/demouser credentials on the captive-portal
```
Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: where is the demouser created?

Comment (extrafu) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: captive portal

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 14]
```diff
@@ -0,0 +1,301 @@
...
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+
+////
+
+    This file is part of the PacketFence project.
+
+    See PacketFence_AeroHIVE_Quick_Installation_Guide-docinfo.xml for
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: This xml file is missing from your pull request.

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 15]
```diff
@@ -0,0 +1,301 @@
...
+
+The instructions are based on version {release_version} of the PacketFence Live Image.
+
+Assumptions
+-----------
+* You will need a USB key with a capacity of at least 4GB;
+* You will need a laptop or a PC from which the PacketFence Live USB key will run from;
+* You will deploy a VLAN enforcement environment according to the customer specifications;
+* Your customer will provide the network equipment (switch, cabled, etc) to interconnect this PoC setup;
+* AeroHIVE AccessPoint is loaded with firmware 6+;
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _firmware 6 or later_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 16]
```diff
@@ -0,0 +1,301 @@
...
+* HiveManager with a version 6+.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _version 6 or later_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 17]
```diff
@@ -0,0 +1,301 @@
...
+Download the application to write the image to the USB key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To write the PacketFence image on a USB key, you'll need to use a third-party application that will provide you with a graphical interface to do so.
+
+Go to https://launchpad.net/win32-image-writer/0.6/0.6/+download/win32diskimager-binary.zip. That will download the necessary application. Once downloaded, extract the zip file.
+
+Write the PacketFence image to the USB key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+First, make sure your USB key is plugged into the PC then open up the folder where you extracted the application and double-click the Win32DiskImager.exe file.
+Once done, simply choose the PacketFence USB image file (unzipped .img file), choose the device on which you want to write and click Write.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Write'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 18]
```diff
@@ -0,0 +1,301 @@
...
+Required interface types for VLAN enforcement:
+
+    Management
+    Registration
+    Isolation
+
+Note that you can only set ONE (1) management interface. This one will work for both in the case you choose both modes.
+
+In our customer scenario, you will create two new vlans on the wired interface (will be eth0 most of the time).
+To do so, click the Add VLAN button besides the wired interface for each of the needed vlan:
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click the 'Add VLAN button'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 19]
```diff
@@ -0,0 +1,301 @@
...
+    IP Address: 192.168.2.1
+    Netmask: 255.255.255.0
+
+Isolation
+
+    Virtual LAN ID: 3
+    IP Address: 192.168.3.1
+    Netmask: 255.255.255.0
+
+Don't forget to also edit the physical interface with the correct management network information by clicking the Edit button next to it.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _clicking the 'Edit' button_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 20]
```diff
@@ -0,0 +1,301 @@
...
+
+    eth0: Management
+    eth0 VLAN 2: Registration
+    eth0 VLAN 3: Isolation
+
+Make sure that those three (3) interfaces are in an Enabled state for the persistance to occur.
+
+We also need to set the Default Gateway which will generally be the gateway of the management network.
+
+Once everything's set, click Continue to proceed with the next step.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Continue'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 21]
```diff
@@ -0,0 +1,301 @@
...
+
+Step 3: Database Configuration
+++++++++++++++++++++++++++++++
+This step will configure the MySQL server needed by PacketFence. Database and schema will be created as well as the necessary user for operations. Root account will also be secured if necessary (set a password and disallow remote login).
+
+Since Debian based MySQL installations are not "secured", you will need to go through it.
+That step is fairly simple to accomplish and is a one time deal.
+
+In the root account credentials section, enter root as Username and click Test. You'll be prompted for a new root password. Choose a password for the MySQL root user and click Save. You can now enter your newly setted password in the root account credentials section and click Test.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Test'_
_click 'Save'_
_click 'Test'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 22]
```diff
@@ -0,0 +1,301 @@
...
+
+Next section of this step is the PacketFence user account on the MySQL server. Simply leave the default pf username here and choose of a password. This one will automatically be set in the PacketFence configuration where you'll be able to retrive it in any case. Once the password entered twice, click Create user.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Create user'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 23]
```diff
@@ -0,0 +1,301 @@
...
+
+Third section will create the database and load the correct schema on it. Simply leave the default and click Create tables and indexes.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Create tables and indexes'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 24]
```diff
@@ -0,0 +1,301 @@
...
+
+You should have Success beside these three section, click Continue.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _have 'Success' beside_
_click 'Continue'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 25]
```diff
@@ -0,0 +1,301 @@
...
+
+Step 4: PacketFence Configuration
++++++++++++++++++++++++++++++++++
+This step will configure the general options of your PacketFence installation. Theses are needed configurations that will most of the time fits customer specifications.
+
+Almost all of the required information here are self-explanatory. The only one that could be confusing is the DHCP Servers section. In this one, enter a comma-delimited list of all the DHCP Server on the customer network so when PacketFence will see DHCP trafic originating from these IPs, no rogue-dhcp alerts will be triggered.
+
+Click Continue once all the field are completed.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click 'Continue'_

Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _all the fields_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 26]
```diff
@@ -0,0 +1,301 @@
...
+
+Step 5: Administration
+++++++++++++++++++++++
+This is the step where you create the administrative user to access the PacketFence Administration Web Interface.
+
+Simply provide the desired username and password, then click Create user.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click 'Create user'._

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc (hunk 2 above)
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click on the 'Configuration' tab_
_the 'Switches' section._

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 27]
```diff
@@ -0,0 +1,301 @@
...
+
+image::docs/images/aerohive-pf-config-1.png[scaledwidth="100%",alt="General switch settings"]
+
+image::docs/images/aerohive-pf-config-role.png[scaledwidth="100%",alt="Roles switch settings"]
+
+image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
+
+Modify your RADIUS secret.
+
+Click Add switch
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click 'Add switch'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 28]
```diff
@@ -0,0 +1,301 @@
...
+
+Connect APs and Controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+First thing to do is to connect the HiveManager. Like you saw in Step 2, make sure to connect the AP on the TRUNK port configured for it.
+
+Provision the HiveManager
+^^^^^^^^^^^^^^^^^^^^^^^^^
+By default, the HiveManager might be empty, add your AP on it via it's serial number. If necessra update the firmware of the HiveManager and the AP.
+
+NOTE: Please refer to the AeroHIVE Quick Devices Start Guide to add devices: http://docs.aerohive.com/330000/docs/help/english/documentation/AerohiveDevicesQuickStart.pdf
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _refer to the 'AeroHIVE Quick Devices Start Guide'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 29]
```diff
@@ -0,0 +1,301 @@
...
+
+NOTE: Please refer to the following guide to learn more about the HiveManager: http://docs.aerohive.com/330000/docs/help/english/documentation/Aerohive_HiveManagerNG_GettingStartedGuide.pdf
+
+Setup the SSIDs
+^^^^^^^^^^^^^^^
+For this guide, you will show you how to configure one open SSID with RADIUS mac filtering, and one secure SSID using WPA2-Enterprise.
+
+Login to your HiveManager using the admin credentials. You should see the Dashboard. Click on the Configure pane, and follow the following steps.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click on the 'Configure' pane_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc (hunk 8 above)
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click 'Ok'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 30]
```diff
@@ -0,0 +1,301 @@
...
++++++++++++++++++++++++++
+On the left menu, click on Advanced Configuration, then Authentication, then AAA Client Settings. We need to add the PacketFence RADIUS server here. Click on the New button, and provide follow this configuration.
+
+image::docs/images/aerohive-radius-pf.png[scaledwidth="100%",alt="RADIUS server configuration"]
+
+Click OK to save the server. You should see the RADIUS server in the list.
+
+Configure the Open SSID
++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those information.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click on 'SSIDs'. Click the 'New' button_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc (hunk 9 above)
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click on 'SSIDs'. Click the 'New' button_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 31]
```diff
@@ -0,0 +1,301 @@
...
+
+Click OK to save the SSID.
+
+Configure the Secure SSID
++++++++++++++++++++++++++
+On the left menu, click on SSIDs. Click the New button, and configure the SSID while following those inforamtion.
+
+image::docs/images/aerohive-ssid-secure.png[scaledwidth="100%",alt="SSID secure configuration"]
+
+Click OK to save the SSID.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click 'OK'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc (hunk 10 above)
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click on the 'Configure' pane, and on 'Advanced Configuration', then 'Management Policies', then 'SNMP Assignements'. Now click on 'New'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc (hunk 12 above)
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _from the 'Configuration' panel, click on 'Networks',_

Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _and click 'Add/Remove'. Configure the 'User Profile' section as follow:_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 32]
```diff
@@ -0,0 +1,301 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To be able to use the "Roles by switch role" you will need to setup a specific RADIUS attribute. To do so in the Configuration pane got to Networks, click on the network you will use, that will bring you to the "Guided configuration" now choose your SSID and click 'Add/Remove' under User Profile configure now this section as follow:
+
+image::docs/images/aerohive-radius-attribute.png[scaledwidth="100%",alt="RADIUS attribute option"]
+
+
+Step 4: Configuration of Windows 7 client for DemoSecure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In `Control Panel\Network and Internet\Manage Wireless Networks`, click on Add.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _click on 'Add'._

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 33]
```diff
@@ -0,0 +1,301 @@
...
+Click Manually create a network profile
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Click 'Manually create a network profile'_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 34]
```diff
@@ -0,0 +1,301 @@
...
+
+Next, change connection settings. On Security tab, click Settings, uncheck Validate server certificate.
+On the same tab click Configure and uncheck Automatically use my Windows logon name and password.
+Return on the Security tab and click on Advanced settings. On 802.1X settings, click on Specify authentication mode and select User authentication.
+On 802.11 settings uncheck Enable Pairwise Master Key (PMK) caching.
+Validate all the modifications and click on close.
+
+Step 5: Test and Demonstrate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Congratulations, you have everything setup and ready!  If your setup is properly configured, you should be able to :
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _you should be able to:_

File: docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc [hunk 35]
```diff
@@ -0,0 +1,301 @@
...
+
+   Network name: DemoSecure
+   Security type WPA2-Entreprise
+   Encryption AES
+
+Next, change connection settings. On Security tab, click Settings, uncheck Validate server certificate.
+On the same tab click Configure and uncheck Automatically use my Windows logon name and password.
+Return on the Security tab and click on Advanced settings. On 802.1X settings, click on Specify authentication mode and select User authentication.
+On 802.11 settings uncheck Enable Pairwise Master Key (PMK) caching.
+Validate all the modifications and click on close.
```
Comment (cgx) [inline docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc]: _Next, change connection settings. On 'Security' tab, click 'Settings', uncheck 'Validate server certificate'.
On the same tab click 'Configure' and uncheck 'Automatically use my Windows logon name and password'.
Return on the 'Security' tab and click on 'Advanced settings'. On 802.1X settings, click on 'Specify authentication mode' and select 'User authentication'.
On 802.11 settings uncheck 'Enable Pairwise Master Key (PMK) caching'.
Validate all the modifications and click on 'Close'._

**Pull request merged**